src/modules/claim/services/claim.service.ts

```diff
@@ -62,7 +62,8 @@ export class ClaimService {
    * @param rq IClaimRequest request
    */
   public async handleClaimEnrolmentRequest(
-    rq: IClaimRequest
+    rq: IClaimRequest,
+    url: string
```

Comment:
I would consider carrying the `redirectUri` name through here
```suggestion
    redirectUri: string
```